projects/program-website/src/app/app.module.ts

```diff
@@ -4,15 +4,26 @@ import { NgModule } from '@angular/core';
 import { AppComponent } from './app.component';
 import { CoreModule } from './core/core.module';
 import { FirebaseModule } from './import/firebase/firebase.module';
+import { AboutComponent } from './about-component/about.component';
+import { ProjectsComponent } from './projects-component/projects.component';
+import { ParticipantsComponent } from './participants-component/participants.component';
+import { ContactUsComponent } from './contact-us-component/contact-us.component';
+import {RouterModule} from '@angular/router';
+import { AppRoutingModule } from './app-routing.module';
```

Comment:
It's recommended to separate third-party libraries imports and imports from the app:
https://angular.io/guide/styleguide#import-line-spacing

Reply:
Handled.

import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import {RouterModule} from '@angular/router';

import { AppComponent } from './app.component';
import { CoreModule } from './core/core.module';
import { FirebaseModule } from './import/firebase/firebase.module';
import { AboutComponent } from './about-component/about.component';
import { ProjectsComponent } from './projects-component/projects.component';
import { ParticipantsComponent } from './participants-component/participants.component';
import { ContactUsComponent } from './contact-us-component/contact-us.component';
import { AppRoutingModule } from './app-routing.module';